pkgs/applications/editors/vim/plugins/moveline-nvim/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  rustPlatform,
+  vimUtils,
+}:
+let
+  version = "2024-07-25";
+  src = fetchFromGitHub {
+    owner = "willothy";
+    repo = "moveline.nvim";
+    rev = "9f67f4b9e752a87eea8205f0279f261a16c733d8";
+    sha256 = "sha256-B4t5+Q4Urx5bGm8glNpYkHhpp/rAhz+lDd2EpWFUYoY=";
+  };
+  moveline-lib = rustPlatform.buildRustPackage {
+    inherit src version;
+    pname = "moveline-lib";
+    cargoHash = "sha256-e9QB4Rfm+tFNrLAHN/nYUQ5PiTET8knQQIQkMH3UFkU=";
```

Comment:
There is not `Cargo.lock` in the repository, so `nix-update` crashes...

Reply:
Ugh... who doesn't check in a Cargo.lock...

Reply:
Even stranger... this plugin thinks there is a Cargo.lock, I go check for the commit and it doesn't show in the list on the repo. I go to the URL for the plugin and it did have a Cargo.lock in that commit. lol